### PR TITLE
Update the course global energy transitions 2022

### PR DESCRIPTION
**Pull request by behnam-zakeri**

### CODE REVIEW

File: lectures/global-entrans-tuwien/index.rst [hunk 1]
```diff
@@ -5,27 +5,26 @@ Global Energy Transitions and Climate Policy @ TU Wien
    :width: 240px
    :align: right
 
-This course is part of the Master's Degree Program in `Environmental Technology & International Affairs`_ 
-held at the Technical University Vienna (TU Wien) since spring 2021.
+This course is part of the Master's degree program in `Environmental Technology & International Affairs`_ 
+held at the Technical University Vienna (TU Wien).
 The course is designed and delivered by `Dr. Behnam Zakeri <https://www.iiasa.ac.at/staff/zakeri>`_,
-including two lectures given by guest speakers Dr. Kavita Surana and Franziska Menten (MSc).
+including a few lectures by guest speakers/lecturers.
```
Comment (danielhuppmann) [inline lectures/global-entrans-tuwien/index.rst]:
```suggestion
including several lectures by guest speakers/lecturers.
```

**Review by danielhuppmann — COMMENTED** [review]
Just one minor suggestion...

File: lectures/global-entrans-tuwien/index.rst (hunk 1 above)
Comment (behnam-zakeri) [reply lectures/global-entrans-tuwien/index.rst]: Thanks @danielhuppmann for the quick feedback. I corrected this.

**Review by danielhuppmann — APPROVED** [review]
Looks good to me, many thanks!

@behnam-zakeri, please check that the page is updated on teaching.ece.iiasa.ac.at and that all links work as expected. Please start a follow-up-PR if there are any problems.

**Pull request merged**